src/Widgets/DataBreadcrumbWidget/DataBreadcrumbWidgetClass.ts

```diff
@@ -0,0 +1,8 @@
+import { provideWidgetClass } from 'scrivito'
+
+export const DataBreadcrumbWidget = provideWidgetClass('DataBreadcrumbWidget', {
+  attributes: {
+    data: 'datalocator',
+    parentData: 'datalocator',
```

Comment:
As an editor I'm confused, what `data` actually refers to. If we already have two `'datalocator'` how about also two named `'datalocator'`? This way we can also name them appropriately (e.g. `labelData: 'datalocator',`) and give them a good editor title as well.